packages/frontend/src/graphql/queries/get-app-connections.ts

```diff
@@ -1,6 +1,6 @@
-import { gql } from '@apollo/client'
+import { graphql } from 'graphql/__generated__'
 
-export const GET_APP_CONNECTIONS = gql`
+export const GET_APP_CONNECTIONS = graphql(`
```

Comment:
wow this is pretty mind blowing, it uses the entire query string as an identifier for the type.